### DISCUSSION TITLE
Coin Yakım Metod ?

### DISCUSSION THREAD

**Discussion by Despair016** [IBAX-io/go-ibax#910] [Soru Cevap - Türkçe] [answered]
Sistem etkinleştirildiğinde jeton yakma işlemlerinin yanma sıklığı ne olacak?
Nasıl bir süreç izleyeceksiniz?
Madencilik Yapılırsa Olası Fiyat Artışlarını Düşünüyor musunuz?

**Comment by Yusufknk** [comment, marked as answer]
İlerleyen zamanlarda yol haritası ile ilgili detaylı bilgi verilecektir.

**Reply by Despair016** [reply]
twetter github aırdrop  kazandım ama hala yatırılmadı yöneticilere söylermıınız  gerekli istedikleri talimatları yaptım
